src/components/content/medication-dispense/helpers/filters.tsx

```diff
@@ -0,0 +1,7 @@
+import { MedicationDispenseModel } from "@/fhir/models";
+
+export const applyMedDispenseFilters = (data: fhir4.MedicationDispense[]) => {
+  const medDispenseModel = data.map((m) => new MedicationDispenseModel(m));
+
+  return medDispenseModel.filter((m) => m.PMARefillNotPickedUp === "PMARefillNotPickedUp");
+};
```

Comment:
nit - I'd write this like so, but it's personal preference, PR approved 👍🏼 
```suggestion
export const applyMedDispenseFilters = (data: fhir4.MedicationDispense[]) => data
  .map((m) => new MedicationDispenseModel(m))
  .filter((m) => m.PMARefillNotPickedUp === "PMARefillNotPickedUp");
```